docs/csharp/language-reference/keywords/in-parameter-modifier.md

```diff
@@ -18,10 +18,10 @@ The `in` keyword causes arguments to be passed by reference. It is like the [ref
 
 [!code-csharp-interactive[cs-in-keyword](../../../../samples/snippets/csharp/language-reference/keywords/in-ref-out-modifier/InParameterModifier.cs#1)]  
 
-The preceding example demonstrates that the `in` modifier is unnecessary at the call site. It is only required in the method declaration.
+The preceding example demonstrates the `in` modifier is usually unnecessary at the call site. It is only required in the method declaration.
 
 > [!NOTE] 
-> The `in` keyword can also be used with a generic type parameter to specify that the type parameter is contravariant, as part of a `foreach` statement, or as part of a `join` clause in a LINQ query. For more information on the use of the `in` keyword in these contexts, see [in](in.md) which provides links to all those uses.
+> The `in` keyword can also be used with a generic type parameter to specify that the type parameter is contravariant, as part of a `foreach` statement, or as part of a `join` clause in a LINQ query. For more information on the use of the `in` keyword in these contexts, see [in](in.md), which provides links to all those uses.
```

Comment:
This is unrelated to your PR, but the in disambiguation topic is in the Query Keywords section in the TOC, which seems like the wrong place.

Reply:
I'm going to create a new issue for this: #4757
That way, this PR stays focused on this single topic.